db/migrate/20170710132935_custom_button_to_classes.rb

```diff
@@ -0,0 +1,144 @@
+class CustomButtonToClasses < ActiveRecord::Migration[5.0]
```

Comment:
`migrate` -> implicit enough from the folder name
everything else is there :wink: :trollface: